HISTORY

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+1.1.2 (2016-06-29)
+------------------
+
+* Pinned to djangocms-attributes-field v0.1.0
+* Fixed model name used in migration
```

Comment:
This is a change that Yakky made in a previous commit, so, when I created this new changelog entry, I just added it.